health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/Filters.js

```
@@ -0,0 +1,273 @@
+              handleCycleFilter({
+                cycle: cycle.code,
+              })
+            }
+          />
+        </div>
+      )}
+      {showDDR && (
+        <div className="filters-input">
+          <div className="mbsm">{t(isNational ? "ES_DSS_STATE" : "ES_DSS_DDR")}</div>
+          <MultiSelectDropdown
+            options={ulbTenants?.ddr && ulbTenants.ddr?.sort((x, y) => x?.ddrKey?.localeCompare(y?.ddrKey))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)